dotcom-rendering/package.json

```diff
@@ -67,8 +67,8 @@
 		"@guardian/bridget": "2.3.0",
 		"@guardian/browserslist-config": "4.0.0",
 		"@guardian/cdk": "49.5.0",
-		"@guardian/commercial": "9.1.0",
-		"@guardian/consent-management-platform": "13.3.1",
+		"@guardian/commercial": "0.0.0-beta-20230622094226",
```

Comment:
Odd version 🤔 Should we be using a beta commercial build?

Reply:
This is just for testing.